extension/api.js

```diff
@@ -58,14 +58,14 @@
 	})();
 
 	window.gitHubNotifCount = function (cb) {
-		var token = GitHubNotify.settings.get('oauthToken');
+		var token = window.GitHubNotify.settings.get('oauthToken');
```

Comment:
You can just add it as a global using the ESLint comment directive. http://eslint.org/docs/user-guide/configuring.html